src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,63 @@
+const GETROCKETS_BEGIN = 'space-travelers/rockets/GETROCKETSBEGIN';
+const GETROCKETS_SUCCESS = 'space-travelers/rockets/GETROCKETSSUCCESS';
+const GETROCKETS_FAILURE = 'space-travelers/rockets/GETROCKETSFAILURE';
+const BASE_URL = 'https://api.spacexdata.com/v3/rockets';
+
+const INITIAL_STATE = {
+  rockets: [],
+  fetching: true,
+};
+
+export function getRocketsBegin() {
+  return {
+    type: GETROCKETS_BEGIN,
+  };
+}
+
+export function getRocketsSuccess(rockets) {
+  return {
+    type: GETROCKETS_SUCCESS,
+    payload: rockets,
+  };
+}
+
+export function getRocketsFailure() {
+  return {
+    type: GETROCKETS_FAILURE,
+  };
+}
+
+export default function reducer(state = INITIAL_STATE, action) {
+  switch (action.type) {
+    case GETROCKETS_SUCCESS:
+      return {
+        rockets: action.payload,
+        fetching: false,
+      };
+    case GETROCKETS_BEGIN:
+      return {
+        ...state,
+        fetching: true,
+      };
+    case GETROCKETS_FAILURE:
+      return state;
+    default: return state;
+  }
```

Comment:
I really liked this logic that you created to handle either when the fetch is not ready or when an error happens.